src/definitions/collections/menu.less

```diff
@@ -635,9 +635,9 @@
   font-weight: @tabularActiveWeight;
 
   margin-bottom: -@tabularBorderWidth;
-  border-bottom: @tabularBorderWidth solid @tabularBackgroundColor;
+  border-bottom: none;
 
-  box-shadow: none;
+  box-shadow: 0px 2*@tabularBorderWidth 0px -@tabularBorderWidth @tabularBackgroundColor;
```

Comment:
This provides a pixel perfect border ![menu border](https://cloud.githubusercontent.com/assets/4712046/5649031/b0798394-969b-11e4-86c8-0c1eeeef99a7.png)